recipes/flit/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.9" %}
+
+package:
+  name: flit
+  version: {{ version }}
+
+source:
+  fn: flit-{{ version}}.tar.gz
+  url: https://github.com/takluyver/flit/archive/{{ version}}.tar.gz
+  sha256: 19f7ef05e0b63b32996e9d7b14a5c8d2d01532628a5413b16015cb5f7e7e44b9
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  binary_has_prefix_files:
```

Comment:
So `conda-build` thinks this is an empty string. It can be fixed by tacking on a `  # [win]` on this line. While a bit ugly, that should hide this key on non-Windows platforms where it contains no elements.